src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+general.organization-did=urn:ngsi-ld:organization:0b03975e-7ded-4fbd-9c3b-a5d6550df7e2
+general.pep=apisix
+quarkus.openapi-generator.codegen.spec.opa_yaml.return-response=true
```

Comment:
Why is this needed at runtime? Isn't the client/server code generated at buildtime?

Reply:
The application.properties are also used at build-time. However, it probably is better to be moved to the maven properties